svc-lighthouse-api/src/main/java/gov/va/vro/abddataaccess/config/AppProperties.java

```diff
@@ -15,7 +15,6 @@ public RabbitMqProperties rabbitmq() {
   }
 
   @Bean
-  @ConfigurationProperties(prefix = "lh")
```

Comment:
Moved to `LighthouseProperties` to enable `@ContextConfiguration(classes = LighthouseProperties.class)`